config/static_test.go

```diff
@@ -65,8 +64,7 @@ var testConfigFullExp = StaticCfg{
 		UpdateCheckFrequency: 14,
 	},
 	Blacklisted: BlacklistedStaticCfg{
-		UseIPms:            true,
-		UseMDL:             true,
```

Comment:
Will the config parser crash if the user still has these fields defined? Would that force users to upgrade their config files?

Reply:
Extra fields are ignored and doesn't cause any problems.